tests/components/webostv/test_device_trigger.py

```diff
@@ -97,11 +97,11 @@ async def test_if_fires_on_turn_on_request(
         blocking=True,
     )
 
-    assert len(calls) == 2
-    assert calls[0].data["some"] == device.id
-    assert calls[0].data["id"] == 0
-    assert calls[1].data["some"] == ENTITY_ID
-    assert calls[1].data["id"] == 0
+    assert len(service_calls) == 3
```

Comment:
Can you explain why all the tests here are now off by one?

Reply:
Because they mow include the call to turn off just above each assert

Reply:
See https://github.com/home-assistant/core/blob/73716ea5297550c591da0aca4f050c597330b6ec/tests/components/webostv/test_device_trigger.py#L93-L98

Reply:
Thanks for the explanation 👍